politeiawww/cmd/piwww/commentcensor.go

```diff
@@ -108,14 +104,13 @@ func (cmd *commentCensorCmd) Execute(args []string) error {
 // commentCensorHelpMsg is the help command message.
 const commentCensorHelpMsg = `commentcensor "token" "commentID" "reason"
 
-Censor a user comment. Requires admin privileges.
+Censor a user comment on a vetted record. Requires admin privileges.
```

Comment:
The command is for either a vetted or unvetted record, with the default being a vetted record.  The help message should reflect that.

Ex.
```
Censor a user comment.  This command assumes the record is a vetted record.  If the record is unvetted, the --unvetted flag must be used.  Requires admin privileges.
``

Apply this same change to all effected help messages.

Reply:
done mate thanks for pointing this out :+1: